Core/src/Surfaces/PlaneSurface.cpp

```diff
@@ -129,7 +129,7 @@ Acts::Polyhedron Acts::PlaneSurface::polyhedronRepresentation(
       // Two concentric rings, we use the pure concentric method momentarily,
       // but that creates too  many unneccesarry faces, when only two
       // are needed to describe the mesh, @todo investigate merging flag
-      auto facesMesh = detail::FacesHelper::cylindricalFaceMesh(vertices, true);
+      auto facesMesh = detail::FacesHelper::cylindricalFaceMesh(vertices);
       faces = facesMesh.first;
       triangularMesh = facesMesh.second;
```

Comment:
```suggestion
      auto [faces, triangularMesh] = detail::FacesHelper::cylindricalFaceMesh(vertices);
```
There's a similar case a few lines up that's not part of the changeset.